### PR TITLE
Remove obsolete acronym expansion

### PR DESCRIPTION
**Pull request by RubenVerborgh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,9 +44,9 @@ For the time being, the present document contains the best approximation of expe
 
 ## Overview
 
-[Solid](https://github.com/solid/solid) (derived from "**so**cial **li**nked
-**d**ata") is a proposed set of conventions and tools for building
-*decentralized social applications* based on [Linked
+[Solid](https://github.com/solid/solid)
+is a proposed set of conventions and tools for building
+*decentralized applications* based on [Linked
 Data](http://www.w3.org/DesignIssues/LinkedData.html) principles. Solid is
```
Comment (csarven) [inline README.md:50]:
```suggestion
Data](https://www.w3.org/DesignIssues/LinkedData) principles. Solid is
```

**Review by csarven — DISMISSED** [review]
This must've been really bugging you to PR on this repo...

(We need to migrate useful issues/PR and archive this repo)

**Comment by RubenVerborgh** [comment]
> This must've been really bugging you to PR on this repo...

I just notice that this is still the first hit when I Google for "Solid specification".
So that must be why so many still think it's about Social LD.

> (We need to migrate useful issues/PR and archive this repo)

Yeah, with a big obsolete warning on top.

**Review by kjetilk — APPROVED** [review]
Yeah, we need to archive, but yes, I can accept this now that it is written :-)

**Review by csarven — APPROVED** [review]

**Pull request merged**